### DISCUSSION TITLE
Trying to use `cython-lint`

### DISCUSSION THREAD

**Discussion by YoSTEALTH** [SublimeLinter/SublimeLinter#1925] [General]
I am trying to use https://github.com/MarcoGorelli/cython-lint to work with SublimeText using SublimeLinter. I have installed `cython-lint` and added
``` json
{
    "linters": {
        "cython-lint":{
                "args": ["--max-line-length=120"],
                "selector": "source.cython",
                "lint_mode": "background",
                "disable": false,
            }
        }
}
```
Nothing seems to be happening! maybe there is more to do that I am missing?

I have also added "executable" and such options ...

**Comment by kaste** [comment]
Yeah, that looks like a nice idea to configure linters just by using JSON.  But we don't have that actually but need a class.  You can start by using our template https://github.com/SublimeLinter/SublimeLinter-template (esp. `linter.py`) but a simple file in your User directory would also work.  Esp. a regex to parse the stdout must be defined in there.
